.circleci/config.yml

```diff
@@ -705,8 +705,6 @@ workflows:
       - docker-build-and-push:
           name: build-discovery-provider
           repo: discovery-provider
-          requires:
-            - test-discovery-provider
```

Comment:
beep

Reply:
I think the intention is to remove this from our circleCI config long term 👀

Reply:
sgtm dawg